src/mjolnir/elevationbuilder.cc

```diff
@@ -99,12 +97,10 @@ std::vector<int8_t> encode_btf_elevation(const std::unique_ptr<valhalla::skadi::
   auto e = encode_elevation(heights, error);
   if (error) {
     double diff = 0;
-    for (size_t i = 0; i < heights.size(); i++) {
-      if (i < (heights.size() - 1)) {
-        auto d = std::abs(heights[i] - heights[i + 1]);
-        diff = d < diff ? diff : d;
-      }
-      LOG_DEBUG("  " + std::to_string(heights[i]));
+    for (size_t i = 1; i < heights.size(); i++) {
+      auto d = std::abs(heights[i] - heights[i - 1]);
+      diff = d < diff ? diff : d;
+      LOG_INFO("  " + std::to_string(heights[i]));
```

Comment:
why the change to INFO though? I remember this being crazy noisy, like 100s mb logs per minute. likely rather the function above, not the btf one, but IMO debug logging is enough for all of them

Reply:
ughh...this is a mistake. sorry.